src/gpg/contact/list.py

```diff
@@ -0,0 +1,38 @@
+import gpg
+import sys
+import os
+from fn.print_key import print_key
+
+
+def ls(contact_list):
```

Comment:
Maybe have a bit more descriptive function name like: `list_contact`, you can do away with comments with this. 
Tip: Your code should be such that it doesn't required comments to describe what it is doing. I think it's pretty clear with the function name. 
Something to read:https://blog.codinghorror.com/coding-without-comments/